model/app/main.py

```
@@ -90,100 +124,73 @@ def store_data(data: dict, index_name: Optional[str] = default_index_name) -> di
     except Exception as e:
         raise Exception(f"Failed to index data: {str(e)}")
    
+from fastapi import Body
+
 """
     This endpoint calls your custom LLM.
 """
 @app.post("/call_llm/")
-def call_llm(prompt: str):
+def call_llm(
+    project_info: ProjectInfo = Body(...), 
```

Ruff (0.8.2)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)